src/CollapsibleField.js

```diff
@@ -211,6 +219,8 @@ class CollapsibleField extends Component {
               onChange={formData => {
                 onBlur(formData);
               }}
+              formContext={{ ...formContext }}
```

Comment:
```
formContext={formContext }
formData={formData }
```
Would the above make more sense?

Reply:
@sneu012 Yeah I think the change you commented makes more sense

Reply:
make sense, Updated to send only Table data